src/assets/i18n/en_US.json

```diff
@@ -2679,5 +2679,17 @@
   "quote.already_in_basket.error": "The quote is already in the cart.",
   "quote.not_found.error": "The quote could not be found.",
   "basket.add_quote.error": "The quote could not be added to the cart.",
-  "quoterequest.not_editable.error": "You have already submitted the quote request. Please <a href=\"javascript:location.reload()\">reload</a> this page to view the changes."
+  "quoterequest.not_editable.error": "You have already submitted the quote request. Please <a href=\"javascript:location.reload()\">reload</a> this page to view the changes.",
+  "checkout.sepa.iban.error.default": "Field is required and should not be empty!",
```

Comment:
pls, use existing error message keys, if possible, e.g. checkout.payment.debit.transfer.account.number.missing.error
if it is not possible for some keys, ask docu team for assistence and provide also German and French translations